cmd/podman/stop.go

```diff
@@ -101,6 +125,10 @@ func stopCmd(c *cliconfig.StopValues) error {
 	}
 	logrus.Debugf("Setting maximum workers to %d", maxWorkers)
 
-	stopErrors, errCount := shared.ParallelExecuteWorkerPool(maxWorkers, stopFuncs)
-	return printParallelOutput(stopErrors, errCount)
+	stopErrors, _ := shared.ParallelExecuteWorkerPool(maxWorkers, stopWorkers)
+	for key, value := range stopErrors {
+		lookupErrors[key] = value
+	}
+
```

Comment:
i think you will want to get to a point where runtime and remote_runtime stop return a list of stopped container ids and a list of errors.